.github/workflows/merge.yml

```diff
@@ -138,70 +138,16 @@ jobs:
           project_id: ${{ secrets.GOOGLE_HOPRASSOCIATION_PROJECT }}
           install_components: beta
 
-      - name: Disable sccache (act-only)
-        if: ${{ env.ACT }}
-        run: |
-          echo "SCCACHE_GHA_ENABLED=false" >> $GITHUB_ENV
-          echo "RUSTC_WRAPPER=" >> $GITHUB_ENV
-
-      - name: Sccache Setup
-        if: ${{ !env.ACT }}
-        uses: mozilla-actions/sccache-action@v0.0.3
-        with:
-          version: "v0.4.0"
-
-      - name: Install Nix
-        uses: cachix/install-nix-action@v20
-        with:
-          github_access_token: ${{ secrets.GITHUB_TOKEN }}
-
-      - uses: cachix/cachix-action@v12
-        if: ${{ !env.ACT }}
-        with:
-          name: hoprnet
-          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
-        continue-on-error: true
-
-      - name: get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "dir=$(pwd)/.yarn/cache" >> $GITHUB_OUTPUT
-
-      - name: restore yarn cache
-        uses: actions/cache@v3
-        id: yarn-cache
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
-
-      - name: restore cargo cache
-        uses: actions/cache@v3
-        with:
-          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock', 'rust-toolchain.toml') }}
-          path: |
-            ~/.cargo/bin/
-            ~/.cargo/registry/index/
-            ~/.cargo/registry/cache/
-            ~/.cargo/git/db/
-            target/
-            .cargo/.package-cache/
-            .cargo/bin/
-            .foundry/bin/
-
-      - name: Install dependencies
-        run: nix develop -c make -j deps-ci
-
       - name: Deploy on GCP
         run: |
           declare base_branch=${{ github.event.pull_request.base.ref }}
-          declare docker_stable_tag
-          if [ "${base_branch}" == "master" ]; then
-            docker_stable_tag=latest
+          if [[ "${base_branch}" == "master" ]]; then
+            ./scripts/deploy.sh rotsee
+          elif [[ "${base_branch}" =~ ^"release" ]]; then
+            ./scripts/deploy.sh "${base_branch/release\/}-dufour"
```

Comment:
Make note that we are hard-coding the network to all release branches here.

Reply:
Yes, this implies that in the future the new PR merged against a release will be deployed in a instanceGroup suffixed with dufour. We will have to change it once we migrate to mont_blanc. But for now, I think that it will work.